projects/vc-api-implementation.md

```diff
@@ -0,0 +1,255 @@
+| Library: credential signatures | DIDKit (https://www.spruceid.dev/didkit) | - Support for many DID methods <br> - Support for both Data Integrity and JOSE VCs (allowing for future JOSE support for this project) <br> - Packaged as libraries for several programming languages, providing future technical agility |
+
+### Implementation details 
+
+#### Key Management 
+
+A VC API implementation which provides the Verifiable Credential issuance (https://w3c-ccg.github.io/vc-api/#issue-credential) or Verifiable Presentation proving (https://w3c-ccg.github.io/vc-api/#prove-presentation) endpoints must have access to private keys (or key services which utilize private keys) in order to create the proofs. However, the API between VC API and a key service is currently out of scope of the VC API specification. 
+
+The project has integrated key material by having a dedicated key module which has HTTP endpoints for the creation, import and export of key material. A tutorial covering key export and import is available here: https://github.com/energywebfoundation/ssi/blob/develop/apps/vc-api/docs/tutorials/key-export-import-tutorial.md.
+
+#### Exchange configuration 
+
+A feature of the VC API specification is support for credential exchanges. This refers to the movement of Verifiable Credentials between actors defined in the Verifiable Credentials Data Model (Issuers, Holders & Verifiers). 
+
+The project has defined a concept of Exchange Definitions in order to specify the configuration of an exchange. Exchange Definitions are documented here: https://github.com/energywebfoundation/ssi/blob/develop/apps/vc-api/docs/exchanges.md#exchange-definitions. The Exchange Definition data model and API endpoints are currently not included in the VC API specification, as noted in this table: https://github.com/energywebfoundation/ssi/tree/develop/apps/vc-api#standard-vs-custom-endpoints. 
+
+However, recent discussions in the VC API working group suggest that APIs for the configuration of exchanges may be included in the specification. An example of these discussions is recorded in this GitHub Issues comment from the 2023-08-08 teleconference: https://github.com/w3c-ccg/vc-api/issues/285#issuecomment-1670228974 
+
+#### Automated Testing 
+
+The project is tested by the two automated test suites which have been developed by the W3C CCG for testing credential issuance (https://github.com/w3c-ccg/vc-api-issuer-test-suite) and credential verification (https://github.com/w3c-ccg/vc-api-verifier-test-suite). The project will strive to pass additional test suites as they become available.
+
+These test suites help demonstrate the potential of clients of the API to interoperate with other VC API implementations.
+Further interoperability testing may be performed in the future as well as relevant opportunities arise.
```

Comment:
@troyronda I added this note about interoperability testing done thus far. Added in this commit https://github.com/openwallet-foundation/project-proposals/pull/12/commits/6da4bb06f492990cda9d871157ab4c1cd6a791dc

Reply:
@jrhender @troyronda, Could we have these suites configured to work with GitHub actions?

Reply:
@skounis I think it is possible. Energy Web been able to execute these test suites on a local machine targeting only our instance of VC API so I believe that it would be possible to do so with GHA. Though this might not be trivial, because the steps to run the suites locally involved cloning the test suite repos and then modifying some of the source code of the suites.